### PR TITLE
V0.9.8 cherry picks

### PR DESCRIPTION
**Pull request by nrb**

### CODE REVIEW

File: pkg/cmd/cli/schedule/create.go [hunk 1]
```diff
@@ -107,13 +107,15 @@ func (o *CreateOptions) Run(c *cobra.Command, f client.Factory) error {
...
-				TTL:                metav1.Duration{Duration: o.BackupOptions.TTL},
+				IncludedNamespaces:      o.BackupOptions.IncludeNamespaces,
+				ExcludedNamespaces:      o.BackupOptions.ExcludeNamespaces,
+				IncludedResources:       o.BackupOptions.IncludeResources,
+				ExcludedResources:       o.BackupOptions.ExcludeResources,
+				IncludeClusterResources: o.BackupOptions.IncludeClusterResources.Value,
+				LabelSelector:           o.BackupOptions.Selector.LabelSelector,
+				SnapshotVolumes:         o.BackupOptions.SnapshotVolumes.Value,
+				TTL:                     metav1.Duration{Duration: o.BackupOptions.TTL},
+				StorageLocation:         o.BackupOptions.StorageLocation,
```
Comment (skriss) [inline pkg/cmd/cli/schedule/create.go]: this shouldn't be in here (guessing this doesn't compile?)

File: docs/aws-config.md [hunk 2]
```diff
@@ -271,3 +296,5 @@ It can be set up for Ark by creating a role that will have required permissions,
 [5]: https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html
 [6]: config-definition.md#aws
 [14]: http://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html
+[20]: faq.md
+[21]: backupstoragelocation-definition.md#aws
```
[the remark this answers is not in the document]
Comment (nrb) [reply docs/aws-config.md]: Removed.

**Comment by nrb** [comment]
I wasn't sure about the beta section, so I kept it. Added #928.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+
+#### Bug Fixes
+  * Discard service account token volume mounts from init containers on restore (#910, @james-powis)
+  * Support --include-cluster-resources flag when creating schedule (#942, @captjt)
+  * Remove logic to get a GCP project (#926, @shuheksha)
+  * Only try to back up PVCs linked PV if the PVC's phase is Bound (#920, @skriss)
+  * Claim ownership of new AWS volumes on Kubernetes cluster being restored into (#801, @ljakimczuk)
+  * Remove timeout check when taking snapshots (#928, @carlisia)
+
 #### [v0.9.8-beta.1](https://github.com/heptio/ark/releases/tag/v0.9.8-beta.1) - 2018-10-10
```
Comment (skriss) [inline CHANGELOG.md]: I would remove the entry for v0.9.8-beta.1; we haven't kept pre-release sections around historically and I don't see any reason to.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+#### [v0.9.8](https://github.com/heptio/ark/releases/tag/v0.9.8) - 2018-10-18
+
+#### Bug Fixes
+  * Discard service account token volume mounts from init containers on restore (#910, @james-powis)
+  * Support --include-cluster-resources flag when creating schedule (#942, @captjt)
+  * Remove logic to get a GCP project (#926, @shuheksha)
```
Comment (skriss) [inline CHANGELOG.md]: typo in the contributor name - it's shubheksha

**Comment by skriss** [comment]
2 small things on CHANGELOG otherwise LGTM

**Comment by nrb** [comment]
I guess I can rebase to squash the BSL removal changes into their relevant commits. I'll do that, but I'd like to leave the rest alone.

**Comment by skriss** [comment]
Also just a reminder to get the CHANGELOG changes into `master` as well.

**Comment by nrb** [comment]
Squashed and https://github.com/heptio/ark/pull/954 made.

**Pull request merged**